hw02-DIYarrayList/src/main/java/ru/otus/DIYArrayList.java

```diff
@@ -0,0 +1,178 @@
+package ru.otus;
+import java.util.*;
+import java.util.function.Consumer;
+
+public class DIYArrayList<T> implements List<T> {
+
+    public T[] elements;
```

Comment:
```suggestion
    private T[] elements;
```

Поля всегда `private`

Reply:
поправил